go/oasis-node/cmd/registry/entity/entity.go

```diff
@@ -392,14 +392,15 @@ func Register(parentCmd *cobra.Command) {
 
 func init() {
 	entityFlags.Bool(cfgAllowEntitySignedNodes, false, "Entity signing key may be used for node registration (UNSAFE)")
+	_ = entityFlags.MarkHidden(cfgAllowEntitySignedNodes)
```

Comment:
Nice catch, should we also feature-gate it behind the `debug.dont_blame_oasis` flag?

Reply:
Done. Not sure how many tests will be broken by this. Let's see...